apps/google-analytics-4/frontend/src/types.ts

```diff
@@ -22,3 +22,30 @@ export interface ServiceAccountKeyId {
   projectId: string;
   clientId: string;
 }
+
+export interface Row {
```

Comment:
nit/sure: Does this need to be exported or will it only ever be used by RunReportResponse and RunReportResponse is what will be used by other components?

Reply:
It is currently only used by `RunReportResponse`, so I will remove the export here.